### PR TITLE
ADEN-10174 Adding player ad info cache in case of video errors

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/ad-products/video/player/porvata/porvata-listener.ts [hunk 1]
```diff
@@ -53,6 +53,11 @@ export class PorvataListener {
 	static LOG_GROUP = 'porvata-listener';
 	static PLAYER_NAME = 'porvata';
 
+	lastKnownAdData = {
```
Comment (Bielik20) [inline src/ad-products/video/player/porvata/porvata-listener.ts:56]: private?

File: src/ad-products/video/jwplayer/helpers/jwplayer-tracking-helper.ts [hunk 2]
```diff
@@ -29,6 +29,12 @@ const trackingEventsMap = {
 type TrackingEvent = keyof typeof trackingEventsMap;
 
 export class JWPlayerTrackingHelper {
+	lastKnownAdData = {
```
Comment (Bielik20) [inline src/ad-products/video/jwplayer/helpers/jwplayer-tracking-helper.ts:32]: ditto

File: src/ad-products/video/player/porvata/porvata-listener.ts (hunk 1 above)
Comment (dmnsobczak) [reply src/ad-products/video/player/porvata/porvata-listener.ts:56]: Added.

File: src/ad-products/video/jwplayer/helpers/jwplayer-tracking-helper.ts (hunk 2 above)
Comment (dmnsobczak) [reply src/ad-products/video/jwplayer/helpers/jwplayer-tracking-helper.ts]: Added.

**Pull request merged**